tasks/move-granule-collections/README.md

```diff
@@ -0,0 +1,45 @@
+# @cumulus/move-granule-collections
+
+This lambda function moves granules between collections in s3 and in postgres
+
+## Message Configuration
+
+For more information on configuring a Cumulus Message Adapter task, see [the Cumulus workflow input/output documentation](https://nasa.github.io/cumulus/docs/workflows/input_output).
+
+### Config
+
+Config object fields:
+
+| field name | type | default | values | description
+| ---------- | ---- | ------- | ------ | -----------
+
+| buckets | object | (required) | | Object specifying AWS S3 buckets used by this task
+| collection | object | (required) | | The cumulus-api collection object
```

Comment:
This is 'target' collection, right? Should this be part of the task input? Since the task can be invoked to move granules to different collections.